integration_testing/features/super-admin/super-admin-windows-installer-translated-strings.feature

```diff
@@ -0,0 +1,47 @@
+Feature: Super admin can see the Windows installer translated strings
+    Super admin can see the Windows installer translated strings during the Kolibri Windows install/uninstall
+
+    Background:
+      Given That the Windows language setting is in <language>
```

Comment:
This means all Scenarios below will start with Windows in Spanish?  However I see English texts (in quotes) instead of Spanish text in the scenarios.  This `Given` should be in the applicable scenario/s only.

Reply:
I translated the texts (in quotes) into Spanish and I added a comment about the equivalent English translation.